script/Arcueid/Day 10/10_00_ARC10_8_2.txt

```diff
@@ -156,12 +161,13 @@ Arcueid coldly asserts that it was all because of a curse.
 [sha:0df050be04b672ca0447ed1a9814fa8a55b2c0fd]{
 -- Page 873, Offset 26801.
 -- 確かにおかしくなったのは金の瞳を見たからだ。
-It's true that I felt strange from looking into that golden eye.
+It's true that I went crazy after looking into that golden eye.
 }
 [sha:db150161e2f772ac6b4d78c3ac1fe7fe972b59ff]{
 -- Page 873, Offset 26802.
 -- けど、それは切っ掛けにすぎない。
-But that's nothing more than an excuse.
+// Slightly liberal
+But it only served to ignite what was already there.
```

Comment:
I honestly say revert

Reply:
Hm, I don't think the original was totally correct either though. I read this as "However, that was only the start/cue", as in it was seeing the eyes that kicked him into gear, rather than that he's actually using them to excuse his actions.

Reply:
no I agree with that interpretation, but that doesn't preclude it being "just an excuse". "But, that [explanation] isn't more than inspired by [her eye]." 
In other words, it's not really a cause so much as a catalyst and I think that's fair to be called "nothing more than an excuse"

Reply:
'Excuse' just doesn't sit right, it feels too much like it's moving blame, which he isn't really. I do like catalyst though, does 
> But it was nothing more than a catalyst

work for you?

Reply:
sure